app/models/prison.rb

```diff
@@ -18,9 +18,9 @@ class Prison < ActiveRecord::Base
     to: :parsed_slot_details
   delegate :finder_slug, to: :estate
 
-  def self.enabled
-    where(enabled: true).order(name: :asc)
-  end
+  scope :enabled, lambda {
+    where(enabled: true).order(name: :asc).includes(:estate)
```

Comment:
Do we always want to eager load estate? Is kind of surprising to see eager loading on a scope that checks a boolean flag.

My preference is to keep scopes simple so that they can easily reused, for example, if we wanted to pluck the id's of enabled prisons we wouldn't be forced to eager load estates